Src/UniWebViewManager.cs

```diff
@@ -0,0 +1,95 @@
+using UnityEngine;
+using System;
+
+public class UniWebViewManager : MonoBehaviour
+{
+    private static UniWebViewManager _instance;
+    public static UniWebViewManager Instance
+    {
+        get
+        {
+            if (_instance == null)
+            {
+                var obj = new GameObject("UniWebViewManager");
+                _instance = obj.AddComponent<UniWebViewManager>();
+                DontDestroyOnLoad(obj);
+            }
+            return _instance;
+        }
+    }
+
+    private UniWebView webView;
+    private string _authUrl;
+
+    public void Init(string authUrl, string urlScheme)
+    {
+        _authUrl = authUrl;
+        GameObject webViewGameObject = new GameObject("UniWebView");
+        webView = webViewGameObject.AddComponent<UniWebView>();
+
+        webView.AddUrlScheme(urlScheme);
+        webView.Load(_authUrl);
+        webView.Frame = new Rect(0, 0, Screen.width, Screen.height);
+    }
+
+    public void StartAuthorization(Action<string> onSuccess, Action<long, string> onError)
+    {
+        Debug.Log($"UniWebViewManager Starting authorization process...");
+
+        if (webView == null)
+        {
+            Debug.Log($"UniWebViewManager is not initialized. Call Init() first.");
+            return;
+        }
+
+        if (!string.IsNullOrEmpty(_authUrl))
+        {
+            webView.Load(_authUrl);
+        }
+
+        webView.Show();
+
+        webView.OnMessageReceived += (view, message) =>
```

Comment:
I think we should use https://docs.uniwebview.com/api/UniWebViewAuthenticationSession.html
instead manually handle message from webview.
auth code should get from callback function
https://docs.uniwebview.com/api/UniWebViewAuthenticationSession.html#onauthenticationfinished